test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2002, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/* @test
+ * @bug 4380543
+ * @key headful
+ * @requires (os.family == "windows")
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary setMargin() does not work for AbstractButton
+ * @run main/manual bug4380543
+*/
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.Insets;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class bug4380543 {
+    static testFrame testObj;
+    static String instructions
+            = "INSTRUCTIONS:" +
+            "\n 1. This is a Windows specific test. If you are not on " +
+            "Windows, press Pass." +
+            "\n 2. Check if the Left insets(margins) is set visually " +
+            "similar to other three sides around Radio Button and CheckBox" +
+            "(insets set to 20 on all 4 sides)." +
+            "\n 3. If Left insets(margins) appear Empty, press Fail, " +
+            "else press Pass.";
+    static PassFailJFrame passFailJFrame;
+
+    public static void main(String args[]) throws Exception {
```

Comment:
```suggestion
    public static void main(String[] args) throws Exception {
```
Avoid C-style array declarations.